test/integration/patients/worklist/worklist.js

```diff
@@ -1880,7 +1880,7 @@ context('worklist page', function() {
 
     cy
       .get('[data-owner-filter-region]')
-      .find('button');
+      .should('not.exist');
```

Comment:
I was a little confused by this test...

If a restricted employee is viewing the `Shared By` worklist, the clinician filter should not be shown, correct?